src/services/users.service.js

```diff
@@ -0,0 +1,29 @@
+const { User } = require('../models/models').models;
+
+class UsersService {
+  getAll() {
+    return User.findAll();
+  }
+
+  getById(id) {
+    return User.findByPk(id);
+  }
+
+  create(user) {
+    return User.create(user);
+  }
+
+  async update(id, user) {
+    await User.update(user, { where: { id } });
```

Comment:
Consider returning the result of the `update` operation to confirm whether the update was successful.